eventlet/wsgi.py

```diff
@@ -554,8 +554,15 @@ def start_response(status, response_headers, exc_info=None):
             # Per HTTP RFC standard, header name is case-insensitive.
             # Please, fix your client to ignore header case if possible.
             if self.capitalize_response_headers:
+                if six.PY2:
+                    def cap(x):
+                        return x.capitalize()
+                else:
+                    def cap(x):
+                        return x.encode('latin1').capitalize().decode('latin1')
```

Comment:
What do you think about reuse the encoding given in the response header (if exist), and why not allow the possibility to pass the encoding as a function param?